pkg/cloud/services/networking/port_test.go

```diff
@@ -536,98 +433,474 @@ func Test_GetOrCreatePort(t *testing.T) {
 	}
 }
 
-func Test_GarbageCollectErrorInstancesPort(t *testing.T) {
+func pointerTo(b bool) *bool {
+	return &b
+}
```

Comment:
We can probably skip this now that we have `k8s.io/utils/pointer`?
```suggestion
```

Reply:
that's a much bigger change that we expect, I've created https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/1896 to bump CAPO using it but this will require a bit of mechanical work that I'm willing to do after I return from vacation.

Reply:
Oh I didn't see any use of `pointerTo` so I thought it was already ready to delete. I probably just missed it or it could be in other files.